llvm/lib/Transforms/Instrumentation/MemorySanitizer.cpp

```diff
@@ -4868,6 +4928,12 @@ struct MemorySanitizerVisitor : public InstVisitor<MemorySanitizerVisitor> {
       break;
     }
 
+    case Intrinsic::x86_vcvtps2ph_128:
```

Comment:
what about x86_vcvtph2ps_128 etc ?
fwiw - it's totally fine to handle them on a separate PR (if they are not 
handled yet)

Reply:
They're not handled yet. The following would need to be added:
1) unit tests
2) generalize the handler function in this patch to handle the return type potentially having fewer vector elements than the input type

Reply:
After more investigation kicked off by a discussion with Dillon internally (b/397863125#comment13): vcvtph2ps is automatically "upgraded" by LLVM into the "fpext" intrinsic, which is handled by MemorySanitizer (but not perfectly).

Reply:
Uploaded test case to https://github.com/llvm/llvm-project/pull/131244, which shows the upgrade to fpext and how fpext is handled by MSan